lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -270,6 +270,13 @@ defmodule LivebookWeb.SessionLive.CellComponent do
     live_component(socket, LivebookWeb.SessionLive.VegaLiteComponent, id: id, spec: spec)
   end
 
+  defp render_output(socket, {:live_widget, :vega_lite, pid}, id) do
+    live_render(socket, LivebookWeb.SessionLive.LiveWidget.VegaLiteLive,
+      id: id,
+      session: %{"id" => id, "pid" => pid}
+    )
+  end
```

Comment:
We should raise a proper error message if we receive a widget we don't know about. this can happen if someone runs a more recent LiveWidget version in an older notebook version.

We should also most likely consider versioning those widgets. For example, what if we do a new vega_lite? We will call it `:vega_lite_v2`? I would also encourage making the third element either a keyword list or a map, so it is easy to add new entries in a backwards compatible fashion (so we don't need to always bump the version).

Reply:
> We should raise a proper error message if we receive a widget we don't know about. this can happen if someone runs a more recent LiveWidget version in an older notebook version.

Good point, we should definitely render something informative!

> We should also most likely consider versioning those widgets. For example, what if we do a new vega_lite?

When the processes communicate they use maps, so it should be pretty easy to extend the widgets with backwards compatibility, but if we were to break something, then it should be ok to use `:vega_lite_v2` or `{:vega_lite, 2}`?

> I would also encourage making the third element either a keyword list or a map, so it is easy to add new entries in a backwards compatible fashion (so we don't need to always bump the version).

My reasoning is that every widget would just be `{type, pid}` and we always ask the pid for the necessary information. Any use case for additional data here?

Reply:
I mean, i don't have a use case, so it can wait. We can always change the format later too... the important is to fail nicely on an output we don't know. :)